README.rst

```diff
@@ -9,7 +9,32 @@ Crate Commoncrawl Importer Plugin
 =================================
 
 This plugin is used to fast import data from `Common Crawl`_ into
-Crate_.
+Crate_. The plugin extends a custom schema ``ccrawl`` to the copy 
+from command which allows to import gzipped common crawl data to
+a table.
+
+The table should have the following schema::
+
+    create table if not exists commoncrawl (
+      ssl boolean primary key, -- http/https
+      authority string primary key, -- xyz.hello.com:123
+      path string primary key, -- /a?d=1#hello
+      date timestamp primary key,
+      week_partition as date_trunc('week', date) primary key,
+      ctype string,
+      clen int,
+      content string INDEX using fulltext with (max_token_length = 40)
+    );
+
+Then the data could be imported, for example::
+
+    copy commoncrawl from 'ccrawl://commoncrawl2014.blob.core.windows.net/cc-main-2014-15/1397609521512.15_CC-MAIN-20140416005201-00000-ip-10-147-4-33.ec2.internal.warc.wet.gz?sp=r&sr=b&sv=2015-04-05&sig=CTRfSf2DhyDPhafaLRhKQGGkseAb5rdapg5hdYz9wrI%3D&se=2016-12-31'
```

Comment:
this url works and it might not be available in the future. we should use an official link.